index.js

```diff
@@ -47,12 +47,23 @@ EncryptedField.prototype.vault = function(name) {
     }
 };
 
-EncryptedField.prototype.field = function(name) {
-    var self = this;
+EncryptedField.prototype.field = function(name, config) {
+    var self            = this;
+    config              = config || {};
+    var hasValidations  = [undefined,null].indexOf(typeof config.validate) === -1;
+
+    config.validate  = config.validate || {};
+
+    var fieldType = (config.hasOwnProperty('type')) ? self.Sequelize.VIRTUAL(config.type) : self.Sequelize.VIRTUAL;
 
     return {
-        type: self.Sequelize.VIRTUAL,
+        type: fieldType,
```

Comment:
Why wouldn't the field be virtual? We don't want the field itself to be saved as it is just a proxy for the decrypted values from the vault. I am concerned that by not having it be virtual, sequelize will try to load the field.